src/server/schemas/contract/index.ts

```diff
@@ -21,46 +21,6 @@ export interface readSchema extends contractSchemaTypes {
   Querystring: Static<typeof readRequestQuerySchema>;
 }
 
-const abiTypeSchema = Type.Object({
```

Comment:
This was all custom written. Removing it since it doesn't seem to match ABI spec. (Source = https://abitype.dev/api/types)